x-pack/plugins/siem/public/components/charts/common.tsx

```diff
@@ -109,3 +121,6 @@ export const getTheme = () => {
   const defaultTheme = isDarkMode ? DARK_THEME : LIGHT_THEME;
   return mergeWithDefaultTheme(theme, defaultTheme);
 };
+
+const kibanaTimezone = chrome.getUiSettingsClient().get('dateFormat:tz');
+export const browserTimezone = kibanaTimezone === 'Browser' ? moment.tz.guess() : kibanaTimezone;
```

Comment:
That's an interesting feature of `moment` timezone with the guess.

I never knew that API existed. That's pretty neat!